services/docker-compose.yml

```diff
@@ -1193,10 +1193,21 @@ services:
     image: ${DOCKER_REGISTRY:-itisfoundation}/storage:${DOCKER_IMAGE_TAG:-master-github-latest}
     init: true
     hostname: "sto-worker-{{.Node.Hostname}}-{{.Task.Slot}}"
+    environment:
+      <<: *storage_environment
+      STORAGE_WORKER_MODE: "true"
+      CELERY_CONCURRENCY: 100
```

Comment:
Please make this an environment variable. I believe this can become even higher.

Reply:
NOTE: this can come in a subsequent PR.